os/sdk-modifying-coreos.md

```diff
@@ -168,7 +168,19 @@ After `build_image` completes, it prints commands for converting the raw bin int
 
 Once you build an image you can launch it with KVM (instructions will print out after `image_to_vm.sh` runs).
 
-If you encounter errors with KVM, verify that it is supported by your CPU and enabled in the BIOS.
+If you encounter errors with KVM, verify that virtualization is supported by your CPU by running `egrep '(vmx|svm)' /proc/cpuinfo`. The `/dev/kvm` directory will be in your host OS when virtualization is enabled in the BIOS.
+
+The `./coreos_production_qemu.sh` file can be found in the `~/trunk/src/build/images/amd64-usr/latest` directory inside the SDK chroot.
+
+#### Boot Options
+
+After `image_to_vm.sh` completes, run `./coreos_production_qemu.sh -curses` to launch a graphical interface to log in to the Container Linux VM.
+
+You could instead use the `-nographic` option, `./coreos_production_qemu.sh -nographic`, which gives you the ability to switch from the VM to the QEMU monitor console by pressing <kbd>CTRL</kbd>+<kbd>a</kbd> and then <kbd>c</kbd>. To close the Container Linux Guest OS VM, run `sudo systemctl poweroff` inside the VM. 
```

Comment:
Would it matter if it's "this" or "the" VM ?

Reply:
the is good